search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/UserTaskFilterTransformer.java

```diff
@@ -104,8 +109,17 @@ public SearchQuery toSearchQuery(final UserTaskFilter filter) {
   @Override
   public List<String> toIndices(final UserTaskFilter filter) {
     if (isCamundaExporterEnabled) {
-      return List.of("tasklist-task-8.5.0_");
+      final String indexName =
+          String.format(
+              TaskTemplate.FULL_QUALIFIED_INDEX_NAME_PATTERN,
+              TaskTemplate.formatIndexPrefix(globalPrefix),
+              TaskTemplate.COMPONENT_NAME,
+              TaskTemplate.INDEX_NAME,
+              TaskTemplate.INDEX_VERSION);
```

Comment:
🔧 ❓ Maybe we can introduce a static method on the `TaskTemplate` that allows us to simple use it without constructing the format on our own.

Example:

```
   final String indexName = TaskTemplate.getIndexNameWithPrefix(globalPrefix);
```

In TaskTemplate:

```
  public static String getIndexNameWithPrefix(final String prefix) {
    return new TaskTemplate(prefix, false).getFullQualifiedName();
   }
```

Just as an idea to reuse the `getFullQualifiedName` but we could also have the format in the static again.